OxfordBodleian/Juel-Jensen/BDLaethe31.xml

```diff
@@ -37,9 +41,566 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
+                        <msItem xml:id="ms_i1">
+                      <locus from="1ra1" to="1ra20"/>
+                            <title type="complete" ref="LIT5920JohnBeg"/>
+                            <note>The text consists of John 1:1–5, followed an elaboration on how, similar to how the Darkness could not overcome the Light
```

Comment:
followed by?

Reply:
I think the reason why I added it was because the LIT record does not specify its contents apart from the biblical quote (and in other cases, maybe it's really just the biblical quote)